src/RealtimeChannel.ts

```diff
@@ -212,7 +216,7 @@ export default class RealtimeChannel {
       throw `tried to subscribe multiple times. 'subscribe' can only be called a single time per channel instance`
     } else {
       const {
-        config: { broadcast, presence },
+        config: { broadcast, presence, private: is_private },
```

Comment:
private is a reserved word which was causing issues

Reply:
gonna make into camelCase